README.md

```diff
@@ -4,6 +4,19 @@ This is the default site package installed with the Neos base distribution. With
 a look into how easy content can be generated and changed.
 It also shows the basic concepts of how to build a site package with custom node types and plugins.
 
+## Use exported Neos.Demo content for setting up a new Neos 9 test project
+
+Please follow these steps after `composer install`:
+
+1) `./flow doctrine:migrate`
+2) `./flow user:create --roles="Neos.Neos:Administrator" <username> <password> <first name> <last name>`
+3) `./flow cr:setup`
+4) `./flow site:create neosdemo Neos.Demo Neos.Demo:Document.Homepage`
+5) `./flow cr:prune`
+6) `./flow cr:import ./Packages/Sites/Neos.Demo/Resources/Private/Content/`
```

Comment:
those commands in this orders do seem logical to some degree but its rather unintuitive. Maybe we can create a site without any nodes?

Reply:
Yes exactly. But this is an issue for neos/neos-development-collection and as we already have some [Onboarding](https://github.com/neos/neos-development-collection/labels/Onboarding) issues to cleanup this process I plan to add it there or even create a new issue 😉.

Reply:
yes thanks ;) just wanted to start the discussion ;)